mongorestore/mongorestore.go

```diff
@@ -35,10 +35,12 @@ type MongoRestore struct {
 
 	TargetDirectory string
 
+	SkipUsersAndRoles bool
```

Comment:
This is a similar situation to the above. The flag in InputOptions is only one consideration when deciding whether to attempt to restore users/roles: https://github.com/llvtt/mongo-tools/blob/3.3.11/mongorestore/metadata.go#L502-L508. We want to skip it even when `restore.NSOptions.DB == ""`